src/js/components/FormField/index.d.ts

```diff
@@ -1,7 +1,10 @@
 import * as React from "react";
-import { Omit, PlaceHolderType, MarginType } from "../../utils";
+import { BoxSideType, BoxSizeType, BoxStyleType, ColorType, Omit, PlaceHolderType, MarginType, SizeType, SideType, StyleType } from "../../utils";
+
+type PositionType = "inner" | "outer";
 
 export interface FormFieldProps {
+  border?: boolean | BoxSideType | {color?: ColorType, position?: PositionType, side?: BoxSideType, size?: BoxSizeType, style?: BoxStyleType} | ({color?: ColorType, position?: PositionType, side?: BoxSideType, size?: BoxSizeType, style?: BoxStyleType})[];
```

Comment:
I believe that FormField is using border from box under the hood because the formfield container (`FormFieldBox`) is defined as a styled Box. See here: https://github.com/grommet/grommet/blob/a6a9ab1e575bb8ff262d2e0b364586e81d892dd8/src/js/components/FormField/FormField.js#L32

Reply:
great! thanks for verifying.